ci/release_setup.md

```diff
@@ -1,14 +1,42 @@
 # Automating releases for a new Knative repository
 
+**Note:** Throughout this document, MODULE is a Knative module name, e.g. `serving`
+or `eventing`.
+
 By using the release automation already in place, a new Knative repository can
 get nightly and official releases with little effort. All automated releases
 are monitored through [TestGrid](http://testgrid.knative.dev).
 
-* **Nightly releases** are built every night from HEAD on the master branch.
-  They are referenced by a date/commit label, in the form `vYYYYMMDD-<commit_short_hash>`.
+* **Nightly releases** are built every night between 2AM and 3 AM (PST), from
+  HEAD on the master branch. They are referenced by a date/commit label, in the
+  form `vYYYYMMDD-<commit_short_hash>`. The job status can be checked in the
+  `nightly` tab in the corresponding repository dashboard in TestGrid. Images
```

Comment:
would be nice to have a Testgrid link

Reply:
Again, per repo, so I can't link; thus the mention of "in the corresponding repository dashboard".